setup.py

```diff
@@ -40,7 +40,7 @@
     install_requires=[
         'thrift',
         'requests',
-        'opentracing>=2.0,<2.1',
+        'opentracing>=2.3,<2.4',
```

Comment:
Why should this require 2.3?  Is a more lenient `opentracing>=2.0,<2.4` not ok?

Reply:
Context vars based scope manager was added in 2.3

Reply:
I see what you mean. Makes sense. I'll only change the upper bound restriction.